<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderStateNodeGUI</class>
 <widget class="QWidget" name="RenderStateNodeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Render State Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>232</width>
          <height>0</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>225</width>
           <height>349</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Textures</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="texture_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Shaders</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="shader_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Backface</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="backface_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Lighting</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="lighting_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Color Material</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="color_material_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Depth Test</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="depth_test_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Wireframe</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="wireframe_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Soft Normals</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="soft_normal_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Hard Normals</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="hard_normal_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Binormals</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="binormal_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Tangents</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="tangent_state">
            <item>
             <property name="text">
              <string>Inherited</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show inherited properties</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>texture_state</tabstop>
  <tabstop>shader_state</tabstop>
  <tabstop>backface_state</tabstop>
  <tabstop>lighting_state</tabstop>
  <tabstop>color_material_state</tabstop>
  <tabstop>depth_test_state</tabstop>
  <tabstop>wireframe_state</tabstop>
  <tabstop>soft_normal_state</tabstop>
  <tabstop>hard_normal_state</tabstop>
  <tabstop>binormal_state</tabstop>
  <tabstop>tangent_state</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>texture_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetTexture(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shader_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetShader(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backface_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetBackface(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lighting_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetLighting(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depth_test_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetDepthTest(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wireframe_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetWireframe(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color_material_state</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RenderStateNodeGUI</receiver>
   <slot>SetColorMaterial(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetTexture(int)</slot>
  <slot>SetShader(int)</slot>
  <slot>SetBackface(int)</slot>
  <slot>SetLighting(int)</slot>
  <slot>SetDepthTest(int)</slot>
  <slot>SetWireframe(int)</slot>
  <slot>SetSoftNormal(int)</slot>
  <slot>SetHardNormal(int)</slot>
  <slot>SetBinormal(int)</slot>
  <slot>SetTangent(int)</slot>
  <slot>SetColorMaterial(int)</slot>
 </slots>
</ui>
